src/play_controller.cpp

```diff
@@ -378,8 +378,11 @@ void play_controller::fire_start()
 	// start event may modify start turn with WML, reflect any changes.
 	gamestate().gamedata_.get_variable("turn_number") = static_cast<int>(turn());
 
-	refresh_objectives();
-	check_objectives();
+	// We might have no teams here, and if so, we don't want to crash
+	if(!get_teams().empty()) {
```

Comment:
@gfgtdf would this be the right fix? With my changes, `display::viewing_side()` attempts vector access, and we don't want to crash at this point if no teams are defined.

Reply:
Nevermind, I just added a check to refresh_objectives since check_objectives already has one.